admin/class-parcelpro-admin.php

```diff
@@ -241,7 +241,7 @@ public function auto_export($order_id)
         if (!$status || $status == '') {
             $allowed_export = $this->settings['auto_export'];
             $method = $this->settings['availability'];
-            $countries = ($method == 'specific') ? $this->settings['countries'] : null;
+            $countries = ($method == 'specific') ? (!empty($this->settings['countries']) ? $this->settings['countries'] : []) : [];
```

Comment:
Misschien zo: $countries = ($method == 'specific') ? ($this->settings['countries'] ?? []) : null